gulp/tasks/build.js

```diff
@@ -172,13 +166,13 @@ function compileIndvES2015ModulesToBrowser() {
         },
         footer: fileName => {
           return isFirebaseApp(fileName) ? `
-          })();` : `
+          })().default;` : `
```

Comment:
Thanks for that link, it has a lot of useful info. Adding `.default` was more of a temporary fix after the changes I made, I want to see if there's a better (proper) way to export/import the namespace.